### PR TITLE
EventsSDK: Create New Playwright GH Action

### PR DESCRIPTION
**Pull request by mtian725**
Ran npm init playwright@latest and edited the github action file to suit our needs.

J=FUS-5879
TEST=auto

Pushed change and verified that new action is listed. Also, verified that the `run_tests` action only runs `npx jest`

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Playwright Tests
+on: [push, pull_request]
+jobs:
+  test:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
```
Comment (mtian725) [inline .github/workflows/playwright.yml:6]: Do you know if this machine costs any different?

https://github.com/yext/analytics/pull/78/files/43fe152ffcf9a9768f99d1c39b764777f6805c56#r1326142457

Comment (mtian725) [reply .github/workflows/playwright.yml:6]: That's what it was defaulted as, and that's what deploy_hold.yml uses. For clarity, what do you mean by costs?

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
```
Comment (mtian725) [inline .github/workflows/playwright.yml:23]: what is retention-days 30 mean? (copied from old PR) @AjayBenno 

https://github.com/yext/analytics/pull/78/files/43fe152ffcf9a9768f99d1c39b764777f6805c56#r1326142207

Comment (mtian725) [reply .github/workflows/playwright.yml:23]: I think it means how long the artifact produced by the run is retained. I'm not seeing anything online that would give me an exact definition. If this is the case, 30 days is definitely overkill. Thoughts on 2 or 3 days?

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (AjayBenno) [reply .github/workflows/playwright.yml:6]: Cool, was just curious.

File: .github/workflows/playwright.yml (hunk 2 above)
Comment (AjayBenno) [reply .github/workflows/playwright.yml:23]: Apparently it doesn't really cost us anymore so 30 seems fine for now.

File: package.json [hunk 3]
```diff
@@ -33,6 +33,8 @@
     "@babel/preset-typescript": "^7.14.5",
```
Comment (AjayBenno) [inline package.json:33]: Curious does the test command run the playwright tests? or can we add a playwright one? Just so npm run tests ends up actually running both playwright and npm tests.

Comment (mtian725) [reply package.json:33]: Nah, it just runs the jest tests. To run the playwright ones we would need to run `npx playwright test`. The suite of tests provided by the init script are the following

```

  npx playwright test
    Runs the end-to-end tests.

  npx playwright test --ui
    Starts the interactive UI mode.

  npx playwright test --project=chromium
    Runs the tests only on Desktop Chrome.

  npx playwright test example
    Runs the tests in a specific file.

  npx playwright test --debug
    Runs the tests in debug mode.

  npx playwright codegen

Comment (mtian725) [reply package.json:33]: As discussed offline, the test command runs both jest and playwright. Actions that currently call npm run test should be changed to just be running the unit tests

Comment (mtian725) [reply package.json:33]: As discussed offline, realized that we can pass down inputs to the reusable workflows. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idwith

**Review by AjayBenno — APPROVED** [review]
LGTM

**Pull request merged**